### ISSUE TITLE
SAP advertisement better to be avoided.

### ISSUE THREAD

**Issue by alexey-milovidov** [cla-assistant/cla-assistant#837]
We are using CLA Assistant for development of [ClickHouse](https://clickhouse.com/)
and SAP advertisement that you have in footer looks way off.

**Comment by Shegox** [comment]
I agree that the current footer might be a bit excessive on the other hand we want as well make sure to get a bit of credit for hosting/providing it. 😄

Maybe something like this would be a better fit, what do you thing? 
<img width="1911" alt="image" src="https://user-images.githubusercontent.com/16856448/154041614-7bbf07af-dac6-4bcd-a9d0-0e93475ac62e.png">

**Comment by mc-hamster** [comment]
That's a really nice footer! Would be great to use that too.

**Comment by schonauchgut** [comment]
dear team @ SAP, dear @shegox,
first of all, thank you for bringing this solution to the open source world. 

Has the footer been further discussed? I really like the suggestion of the new version. You may even want to add some more clarity, stating "CLA Bot solution is provided with <3 from the Open Source Team at SAP" - otherwise the content of self-hosted CLA Bot solutions may be associated with you. What are your thoughts?

Constantin Hoferer, constantin.hoferer@mercedes-benz.com, on behalf of Mercedes-Benz Tech Innovation GmbH, [Provider Information](https://github.com/mercedes-benz/daimler-foss/blob/master/PROVIDER_INFORMATION.md)

**Comment by Shegox** [comment]
Hi all, 

sorry for the long wait on this. I missed the 👍 on the proposal and forgot about. 

@schonauchgut very good point you bring up with self-hosted instances. 
After looking into a bit, I will now go ahead and add the footer as a configuration option. 
Per default it will say something like `Powered by the open source CLA-assistant project` and on cla-assistant.io then something like `Provided with <3 from the Open Source Team at SAP`.

**Comment by Shegox** [comment]
short update: I removed now the logo and replaced it with a more generic statement  (`Powered by the open source CLA-assistant project`; see #870). 

For the moment being it will not be configurable, but you're free to change it for your own deployment as you like. 

On a side note, I got now the time to look at the whole of the setup again and most likely we will end up rebuilding most of the UI components in a newer framework and as part of that rebuild we can/will make a lot of these things more easily user configurable.
